diff --git a/node_modules/astro/dist/content/utils.js b/node_modules/astro/dist/content/utils.js
index 15b437e..7706a25 100644
--- a/node_modules/astro/dist/content/utils.js
+++ b/node_modules/astro/dist/content/utils.js
@@ -141,7 +141,7 @@ function getContentEntryIdAndSlug({
   const relativePath = getRelativeEntryPath(entry, collection, contentDir);
   const withoutFileExt = relativePath.replace(new RegExp(path.extname(relativePath) + "$"), "");
   const rawSlugSegments = withoutFileExt.split(path.sep);
-  const slug = rawSlugSegments.map((segment) => githubSlug(segment)).join("/").replace(/\/index$/, "");
+  const slug = rawSlugSegments.map((segment) => githubSlug(segment, true)).join("/").replace(/\/index$/, "");
   const res = {
     id: normalizePath(relativePath),
     slug
